Adds the Serverless changelog for November 10 2025 (#3901)
elastic/docs-content: release-notes/elastic-cloud-serverless/index.md

# {{serverless-full}} changelog [elastic-cloud-serverless-changelog]
Review the changes, fixes, and more to {{serverless-full}}.

## November 10, 2025 [serverless-changelog-11102025]

### Features and enhancements [serverless-changelog-11102025-features-enhancements]

> * Enables {{ml}} to manage anomaly detection results indices in Elasticsearch Serverless [#136065]({{es-pull}}136065)

Reviewer: The "in Elasticsearch Serverless" IMO implies that this only applies to the Elasticsearch project, which seems unlikely. I think the original summary from the changelog is better:
```suggestion
* Nightly maintenance for anomaly detection results indices to keep to manageable size [#136065]({{es-pull}}136065)
```

* Adds the ability to preview index requests in Transform for Elasticsearch Serverless [#137455]({{es-pull}}137455)
* Allows field capabilities to span across Elasticsearch Serverless projects [#137530]({{es-pull}}137530)
* Improves ES|QL performance by skipping unnecessary query plan diff calculations in Elasticsearch Serverless [#137721]({{es-pull}}137721)

> * Passes the {{es}} version in the EIS inference request header in Elasticsearch Serverless[#137643]({{es-pull}}137643)

Reviewer: IMO this is worth spelling out:
```suggestion
* Passes the {{es}} version in the Elastic Inference Service (EIS) request header [#137643]({{es-pull}}137643)
```

* Moves visualization configuration settings, including appearance, titles and text, axis, and legend to a flyout panel in **Lens** [#240804]({{kib-pull}}240804)
* Supports subqueries in the Discover pretty printer [#241473]({{kib-pull}}241473)
* Adds context-aware autocomplete for Discover subqueries with nesting restrictions [#241912]({{kib-pull}}241912)
* Adds subquery support for columns after and validation in Discover [#241567]({{kib-pull}}241567)
* Adds support for Discover subqueries in FROM clauses across tools [#242166]({{kib-pull}}242166)
* Enables users to view the SLO associated with a burn rate rule on the rule details page in Elastic Observability Serverless [#240535]({{kib-pull}}240535)
* Exposes 'sampling_rate' agent central config options to users in Elastic Observability Serverless [#241908]({{kib-pull}}241908)
* Makes the Elastic logo open a custom home page in solution view [#241571]({{kib-pull}}241571)
* Enforces the `object_src 'none'` directive in Kibana's Content Security Policy [#241029]({{kib-pull}}241029)
* Adds origin configuration options for authentication providers [#239993]({{kib-pull}}239993)

> * Adds the ability to cancel Machine Learning file uploads [#241297]({{kib-pull}}241297)

Reviewer 2: ```suggestion
* Adds the ability to cancel {{ml}} file uploads [#241297]({{kib-pull}}241297)
```

* Improves display of long field values in Data Visualizer top values list [#241006]({{kib-pull}}241006)
* Adds a temperature parameter to Inference AI, and OpenAI, Bedrock, and Gemini connectors [#239806]({{kib-pull}}239806)
* Adds support for custom headers in the OpenAI integration [#238710]({{kib-pull}}238710)

> * Fixes public update space APIs [#242136]({{kib-pull}}242136)

Reviewer 2: ```suggestion
* Fixes public Update spaces APIs [#242136]({{kib-pull}}242136)
```

* Improves layout for custom inference endpoints [#241779]({{kib-pull}}241779)
* Displays field data types in the Processing table and step editor [#241825]({{kib-pull}}241825)
* Adds timezone and locale parameters to Streamlang [#241369]({{kib-pull}}241369)

> * Displays field data types in the partitioning UI [#242134]({{kib-pull}}242134)

Reviewer 2: It’s capitalized on line 42 and this refers to the same Partitioning tab in Streams, I think.
```suggestion
* Displays field data types in the Streams Partitioning UI [#242134]({{kib-pull}}242134)
```

* Adds autocomplete for field values in Streams Partitioning and Processing tabs [#241119]({{kib-pull}}241119)
* Hides document match filter controls for users without manage privileges [#242119]({{kib-pull}}242119)

### Fixes [serverless-changelog-11102025-fixes]

> * Fixes feature display order when using explain in LTR [#137671]({{es-pull}}137671)

Reviewer: I think it's worth spelling out this acronym
```suggestion
* Fixes feature display order when using explain in Learning To Rank (LTR) [#137671]({{es-pull}}137671)
```

* Fixes an issue where missing geotile buckets caused errors in Transform [#137476]({{es-pull}}137476)

> * Ensures ES|QL full text functions accept `null` values as field parameters in Elasticsearch Serverless  [#137430]({{es-pull}}137430)

Reviewer: Ditto previous comments about "Elasticsearch Serverless":
```suggestion
* Ensures {{esql}} full text functions accept `null` values as field parameters [#137430]({{es-pull}}137430)
```

* Fixes a missing attribute issue in ES|QL full text functions in Elasticsearch Serverless [#137395]({{es-pull}}137395)
* Fixes a bug in `RankDocRetrieverBuilder` when `from` is set to the default -1 value in Elasticsearch Serverless [#137637]({{es-pull}}137637)
* Prevents use-after-close errors in async search by making `MutableSearchResponse` reference-counted in Elasticsearch Serverless [#134359]({{es-pull}}134359)
* Removes early phase failures during batched search execution in Elasticsearch Serverless [#136889]({{es-pull}}136889)
* Improves SQL validation errors by providing more descriptive exception messages in Elasticsearch Serverless [#137560]({{es-pull}}137560)

> * Correctly accounts for additional settings providers when determining data stream effective settings in Elasticsearch Serverless [#137407]({{es-pull}}137407)

Reviewer: ```suggestion
* Correctly accounts for additional settings providers when determining data stream effective settings [#137407]({{es-pull}}137407)
```

* Adds proxy SSL options for download sources [#241115]({{kib-pull}}241115)
* Ensures Fleet policy name uniqueness is enforced consistently across spaces [#239631]({{kib-pull}}239631)
* Shows warnings on the sync integrations UI when referencing other entities [#241623]({{kib-pull}}241623)
* Escapes special characters when creating ESQL queries for **Lens** charts in Elastic Observability Serverless [#241662]({{kib-pull}}241662)
* Fixes "Values" dropdown display on smaller screens in Elastic Observability Serverless [#241812]({{kib-pull}}241812)
* Excludes stale SLOs from group-by statistics in Elastic Observability Serverless [#240077]({{kib-pull}}240077)
* Fixes missing `EngineMetadata.type` in generic entity popovers in Elastic Security Serverless [#239661]({{kib-pull}}239661)
* Sanitizes lookup names when creating indices in Elastic Security Serverless  [#240228]({{kib-pull}}240228)

> * Supports multiple values in IOC flyout table tab in Elastic Security Serverless  [#236110]({{kib-pull}}236110)

Reviewer 2: ```suggestion
* Supports multiple values in IOC flyout table tab in Elastic Security Serverless [#236110]({{kib-pull}}236110)
```
